turf/README.md

```diff
@@ -0,0 +1,10 @@
+# Turf
+
+Taken from here https://turfjs.org
```

Comment:
Again, where does the external code comes from, what release, how to update it...?